src/components/Quote.jsx

```diff
@@ -52,9 +52,9 @@ function Quote() {
       <h2>
         {quote}
         <span className="author">
-          &nbsp;(
+          &nbsp;
+          ~
           { author}
-          )
         </span>
```

Comment:
please remove the console.log statement in line 38 and perform an action if there is a potential error in order to make your code follow best practices.

Reply:
remove console.log and add error message if fetch API got an error